app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -88,20 +88,22 @@
           <TextView
             android:id="@+id/chapter_summary"
             android:layout_width="0dp"
-            android:layout_height="0dp"
+            android:layout_height="wrap_content"
             android:layout_marginStart="8dp"
             android:layout_marginTop="8dp"
             android:layout_marginEnd="8dp"
+            android:layout_marginBottom="4dp"
             android:ellipsize="end"
             android:fontFamily="sans-serif"
-            android:maxLines="4"
+            android:maxLines="3"
```

Comment:
This is incorrect, this file is for portrait mode and as mentioned earlier in portrait mode we have to keep it `4` and in landscape mode we have to keep it `3`

Reply:
@rt4914 in the above comment you mentioned in reverse order